sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -249,6 +249,13 @@ def raise_with_traceback(self):
             self.__traceback__ = self.exc_traceback
             raise self
 
+    def __repr__(self):
+        class_name = self.__class__.__name__
+
+        return "{}(message={})".format(
+            class_name, self.message
+        )[:1024]
```

Comment:
How does this compare the output of `__repr__` for the default Exception? How does it compare to the output of `str()`?
Just curious :)